src/main/java/edu/wpi/cscore/VideoSource.java

```diff
@@ -180,6 +180,24 @@ public boolean setFPS(int fps) {
     return CameraServerJNI.setSourceFPS(m_handle, fps);
   }
 
+  /**
+   * Get the actual FPS.
+   * CameraServerJNI#setTelemetryPeriod() must be called for this to be valid.
```

Comment:
This method should throw `IllegalStateException` if telemetry isn't enabled, instead of failing silently. Alternatively, the native code could throw an exception and the JNI could wrap it in an `IllegalStateException` for Java users.